docs/Displaying a User Interface/Subscribing to System Buttons/index.md

```diff
@@ -1,16 +1,31 @@
 # Subscribing to System Buttons
 Subscribe buttons are used to detect changes to hard buttons located in the car's center console or steering wheel. You can subscribe to the following hard buttons:
 
-| Button  | Template | Button Type |
-| ------------- | ------------- | ------------- |
-| Play / Pause | media template only | soft button and hard button |
-| Ok | media template only | soft button and hard button |
-| Seek left | media template only | soft button and hard button |
-| Seek right | media template only | soft button and hard button |
-| Tune up | media template only | hard button |
-| Tune down | media template only | hard button |
-| Preset 0-9 | any template | hard button |
-| Search | any template | hard button |
+| Button  | Template |
+| ------------- | ------------- |
+| Play / Pause | Media only |
+| Ok | Media only |
+| Seek left | Media only |
+| Seek right | Media only |
+| Tune up | Media only |
+| Tune down | Media only |
+| Preset 0-9 | Any |
+| Search | Any |
+| Center Location | NavigationFullscreenMap only |
```

Comment:
I feel like there needs to be a comment either above the table saying that the  `NavigationFullscreenMap` will be set automatically if the `appType` is `NAVIGATION`. And the media template will be set automatically if the `appType` is`MEDIA`.